alembic/env.py

```diff
@@ -0,0 +1,81 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+import sys
+from os.path import relpath, abspath, join
+from os import getcwd
+sys.path.insert(0, join(abspath(getcwd()), "app"))
+from models import *
+target_metadata = User.metadata
```

Comment:
does `User.metadata` all information for tables and schmea rules?

Reply:
Yes, all the models need to be imported, but only one models metadata needs to be passed

Reply:
OK. Since you import all models in __init__.py shouldn't just `import models` work?

Reply:
It does, changed it to import models